docs/overview.md

```diff
@@ -134,6 +134,21 @@ The override process has two parts:
 1.  Override your template tag with a mock implementation
 2.  Define fake result for your tag in a `yaml` file
 
+
+### Providing a default value for template tags
+To provide a default for a template tag, you need to provide a keyword argument default when overriding your tag.
+
+```
+from pattern_library.monkey_utils import override_tag
+
+override_tag(register, 'a_tag_name', default="https://potato.com")
+```
+
+This default is used for any tag that's not passed its own context, allowing specificity for those elements that need it while preventing the tag from breaking when it's not structural to the component.
+
+#### limitation
+Currently the default override is limited to direct references only i.e. {% a_tag_name page.url %} and not {% a_tag_name page.url as variable_name %} 
```

Comment:
I'm not sure about the wording here. What's actually not supported is modifying context, so it might be better to say something along the lines of "Only providing a default for the output of the tag is supporting, it's not possible to modify context like <example of simpletag `as`>".

Reply:
Fair point, thanks.